src/db/migrations/24-add-crdt-messages.ts

```diff
@@ -0,0 +1,33 @@
+import { Knex } from 'knex';
+
+import { Table } from '../db';
+import { tableNameToViewName, updateViews } from '../migration-helpers';
+
+export const up = async (knex: Knex) => {
+  await knex.schema.createTable(Table.crdtMessages, table => {
+    table.string('timestamp').primary()
+    table.string('entityId');
+    table.enu('table', Object.values(Table));
+    table.string('column')
+    table.string('value');
+  })
+
+  await knex.schema.createTable(Table.processedMessages, table => {
+    table.string('messageId').primary();
+    table.enu('table', Object.values(Table));
+    table.string('column');
+    table.string('entityId');
+    table.string('processedAt');
+  })
```

Comment:
Idea:

We make this table table/column/entityId centric, ie record-centric and it is used to track the latest update to a given column on a given record, rather than to track anything about the messages that caused the update